.github/workflows/ci.yml

```diff
@@ -91,6 +91,42 @@ jobs:
           start: npm run dev:standalone
           wait-on: "http://localhost:3000/search"
 
+  api-build:
+            name: Build API
+            runs-on: ubuntu-latest
+
+            env:
+              api-directory: ./api
+
+            strategy:
+                matrix:
+                  dotnet-version: [ '9.0.x' ]
+
+            defaults:
+              run:
+                working-directory: ${{ env.api-directory }}
+
+            steps:
+              - name: Checkout source
+                uses: actions/checkout@v4
+
+              - name: Setup dotnet ${{ matrix.dotnet-version }}
+                uses: actions/setup-dotnet@v3
+                with:
+                  dotnet-version: ${{ matrix.dotnet-version }}
+
+              - name: Display dotnet version
+                run: dotnet --version
```

Comment:
Looks like the "Setup dotnet" step already prints the dotnet version being used, so is this redundant?
![Screenshot 2024-12-05 at 16 39 21](https://github.com/user-attachments/assets/a2d5ce31-a739-4ec1-975f-9c96e43bca4a)

Reply:
Removed.